components/PaymentOutcome/payment-outcome.module.css

```diff
@@ -65,3 +65,31 @@
   border: 10px;
   padding: 0.5rem;
 }
+
+
+.pay_new_btn {
+  background-color: transparent;
+  color: rgb(83, 111, 242);
+  border: 2px solid rgb(83, 111, 242);
+  border-radius: 10px;
+  width: 270px;
+  height: 50px;
+  margin-bottom: 1rem;
+}
+
+
+
+.reference{
+  word-break: break-all;
+  width:270px;
+  margin:auto
+}
+
+.backdrop{
+  opacity: 1 !important;
```

Comment:
can we avoid using important in our CSS? We can use it for debugging for I don't think its recommended using `!important` for prod CSS except where necessary. Please look for the underlying CSS issue and fix it

Reply:
Hi Theo, thank you for this feedback, I had to add !important because the overlay for the modal being used to display the transaction receipt has an opacity even when I set the opacity of the modal's backdropclassName to 1. 

It was the recommended approach in this stackoverflow thread: https://stackoverflow.com/questions/28096346/how-to-change-background-opacity-when-bootstrap-modal-is-open

I'll remove important from the other stylings I used, but I'm not sure I can for this backdrop one as it is required because of Bootstrap